src/frontend/sharedComponents/DeviceCard.tsx

```diff
@@ -44,30 +54,60 @@ export const DeviceCard = ({
       throw new ExhaustivenessError(deviceConnectionStatus);
   }
 
+  const {navigate} = useNavigationFromRoot();
+  const {formatMessage} = useIntl();
+
   return (
     <TouchableOpacity
       disabled={!onPress || isDisconnected}
       onPress={() => (onPress ? onPress() : {})}
-      style={[styles.container, style]}>
-      <DeviceNameWithIcon
-        name={name}
-        deviceConnectionStatus={deviceConnectionStatus}
-        thisDevice={thisDevice}
-        deviceType={deviceType}
-        deviceId={deviceId}
-        iconSize={75}
-      />
+      style={[style]}>
```

Comment:
nit: no need for the array now

```suggestion
      style={style}>
```

Reply:
https://github.com/digidem/comapeo-mobile/pull/674/commits/da4aa3596be535cd638ec9a58cb90a086605bbb0